lib/algolia/transport/request_options.rb

```diff
@@ -34,7 +34,7 @@ def create(opts = {})
       def add_headers(opts = {})
         unless opts[:headers].nil?
           opts[:headers].each do |opt, value|
-            @headers[opt.to_sym] = value
+            @headers[opt.to_s] = value
```

Comment:
Why do we cast to a string instead of a symbol?

Reply:
The format of the headers will always be string anyway, because of headers like 'X-Algolia-XXX'

Reply:
If it's always a string, we don't need to add the `.to_s` right? 🙂

Reply:
I meant they're supposed to be always string, so it's a way to ensure it's the case